site/content/en/reference/_index.md

```diff
@@ -93,6 +93,34 @@ By default, kpt will use the builtin schema.
   The path to an OpenAPI schema file. The default value is ./openapi.json
 ```
 
+### Annotations
```

Comment:
Currently annotations are described here:
https://github.com/kubernetes-sigs/kustomize/blob/master/cmd/config/docs/api-conventions/config-io.md

Either:
1. Include other annotation here
2. Link to the upstream doc

I vote for #1 for kpt docs to be self-contained.

Reply:
Added information about the two remaining annotations. They are not as useful though for the normal kpt user, as they are mostly for internal use.